src/Megamix/Region.cpp

```diff
@@ -80,6 +80,39 @@ namespace Region {
         }
     }
 
+    std::vector<u32> MuseumRowsInfoAddresses() {
```

Comment:
add a comment like `// Extra museum rows patch` or whatever you wanna call it, in order to separate this from the RHMPatch stuff